compiler-core/src/ast.rs

```diff
@@ -64,12 +64,12 @@ impl TypedModule {
 /// ```
 ///
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct TargettedDefinition {
+pub struct TargetedDefinition {
```

Comment:
Please revert this, we use traditional English rather than USA English spelling in this repo.